### PR TITLE
Possible fix for issue #2965

### PR DESCRIPTION
**Pull request by shrey183**
#### Reference Issue

Fixes issue #2965 

#### What does this implement/fix? Explain your changes.
Changes made to the the cleanup function in MolStandardize.cpp and catch_tests.cpp.

#### Any other comments?
None.

### CODE REVIEW

File: Code/GraphMol/MolStandardize/MolStandardize.cpp [hunk 1]
```diff
@@ -18,16 +18,16 @@
...
 #include <GraphMol/SmilesParse/SmilesParse.h>
-using namespace std;
 
+using namespace std;
 namespace RDKit {
 namespace MolStandardize {
 const CleanupParameters defaultCleanupParameters;
 
-RWMol *cleanup(const RWMol &mol, const CleanupParameters &params) {
+RWMol *cleanup(const RWMol &mol, const CleanupParameters &params)
```
Comment (greglandrum) [inline Code/GraphMol/MolStandardize/MolStandardize.cpp]: This formatting is different from what we normally use. Please switch it back .

File: Code/GraphMol/MolStandardize/MolStandardize.cpp [hunk 2]
```diff
@@ -37,6 +37,8 @@ RWMol *cleanup(const RWMol &mol, const CleanupParameters &params) {
   RWMol *reionized = MolStandardize::reionize(normalized.get(), params);
   MolOps::assignStereochemistry(*reionized);
 
+  reionized->updateProps(m);
```
Comment (greglandrum) [inline Code/GraphMol/MolStandardize/MolStandardize.cpp]: I think it's worth putting a comment in front of this explaining what it's doing.

File: Code/GraphMol/MolStandardize/MolStandardize.cpp (hunk 1 above)
Comment (shrey183) [reply Code/GraphMol/MolStandardize/MolStandardize.cpp]: The problem is with the bracket, right? I am guessing you want it to be on the same line.

Comment (greglandrum) [reply Code/GraphMol/MolStandardize/MolStandardize.cpp]: Right

Comment (greglandrum) [reply Code/GraphMol/MolStandardize/MolStandardize.cpp]: Right

**Comment by greglandrum** [comment]
Thanks @shrey183!

**Pull request merged**